nexus/reconfigurator/planning/src/mgs_updates/host_phase_1.rs

```diff
@@ -420,6 +427,27 @@ pub(super) fn try_make_update(
         return Ok(MgsUpdateOutcome::NoUpdateNeeded);
     }
 
+    // Make sure the board we're targetting doesn't contain any zones that are
+    // unsafe to shut down
+    if unsafe_zone_boards.contains_key(baseboard_id) {
+        let zone_str = match unsafe_zone_boards.get(baseboard_id) {
+            Some(board) => {
+                let mut s = String::new();
+                for (zone_id, zone) in board {
+                    s.push_str(&format!("* {}: {}", zone_id, zone));
```

Comment:
Ugh, I wasn't able to get this printing nicely yet. The "\n" doesn't work here. I'll keep trying 😄